### PR TITLE
[CCI] Adding second command to Install yarn step in Developer Guide

### PR DESCRIPTION
**Pull request by kappassov**
### Description
Currently, Developer Guide has only one command listed in `yarn` installation step: `npm i -g corepack` . Writing only this one command seems to be incomplete and might be confusing for newcomers. Moreover, current description requires making extra step by going to the official installation docs without even trying Developer Guide's instructions.
 
### Issues Resolved
Adding the following command `corepack prepare yarn@<version> --activate` to the `yarn` installation paragraph will fulfill all minimum required steps to install `yarn`, and might help to avoid extra visiting `yarn` documentation.

Screenshot_20230319_212213

 
### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: DEVELOPER_GUIDE.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ If it's the only version of node installed, it will automatically be set to the
 $ npm i -g corepack
 ```
 
+```bash
+$ corepack prepare yarn@<version> --activate
```
Comment (joshuarrrr) [inline DEVELOPER_GUIDE.md:71]: let's just combine them in a single code block.

```suggestion
$ corepack prepare yarn@<version> --activate
```

File: DEVELOPER_GUIDE.md [hunk 2]
```diff
@@ -227,15 +233,15 @@ This method can also be used to develop against the [full distribution of OpenSe
...
 changes to the default [`opensearch_dashboards.yml`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml#L25-L72) in order to connect to OpenSearch.
 
 ```yml
-opensearch.hosts: ["https://localhost:9200"]
-opensearch.username: "admin" # Default username on the docker image
-opensearch.password: "admin" # Default password on the docker image
+opensearch.hosts: ['https://localhost:9200']
+opensearch.username: 'admin' # Default username on the docker image
+opensearch.password: 'admin' # Default password on the docker image
 opensearch.ssl.verificationMode: none
```
Comment (joshuarrrr) [inline DEVELOPER_GUIDE.md:245]: I'd skip this change - I like most of the other markdown linting changes, but these examples in the yml file use double quotes.

File: DEVELOPER_GUIDE.md (hunk 1 above)
Comment (joshuarrrr) [reply DEVELOPER_GUIDE.md:71]: Don't we also need to include how to get the version?

```
Take a look at the [latest Yarn release](https://github.com/yarnpkg/berry/releases/latest), note the version number, and run:
```

**Review by joshuarrrr — COMMENTED** [review]
Excellent find, thanks!

If we're going to add that, I think we also want to include the link to the yarn version. One other small nit - otherwise looks great!

File: DEVELOPER_GUIDE.md (hunk 1 above)
Comment (kappassov) [reply DEVELOPER_GUIDE.md:71]: Thank you! Great suggestions, working on them.

File: DEVELOPER_GUIDE.md (hunk 2 above)
Comment (kappassov) [reply DEVELOPER_GUIDE.md:245]: Changed back.

**Review by joshuarrrr — APPROVED** [review]

**Comment by joshuarrrr** [comment]
I set DCO to pass, as it was a false positive from a rebase commit.

**Review by abbyhu2000 — APPROVED** [review]
This LGTM, thanks!

**Pull request merged**